mozilla/network-pulse-api
# Add entry count to profile routes

Adds an entry count for published, created, and favorited entries for profiles. Also added prefetching to the /myprofile route to reduce from 21 queries to 9.

To test:
1. Test the `api/pulse/profiles/<id>` routes to make sure that they include the `entry_count` object
2. Test the `api/pulse/myprofile` route to make sure that it also includes the entry count and is running less than 10 queries.

@mmmavis would you be able to test the routes you need for the profile view to make sure they have all the information you need?
